types/src/front/assistant/agent.ts

```diff
@@ -152,3 +152,21 @@ export type AgentConfigurationType = LightAgentConfigurationType & {
   // performed (potentially NoOp eg autoSkip above).
   action: AgentActionConfigurationType | null;
 };
+
+export function isGlobalAgentConfiguration(
```

Comment:
IMHO we don't need those 3 functions, the scope test is clear enough and short enough; available to discuss if needed

Wrapping lower level logic in functions is fine if either it dries enough, or it makes clearer enough, here IMHO not the case

Reply:
the varying names are confusing (isglobal, then isagentinworkspace, then isagentPublished

Reply:
I went with this approach following our discussion from yesterday, were I feel like we the scope is slightly changing with the ongoing refactoring. But I agree with the "dry" point and I'm happy to remove it.

Reply:
I'm happy to remove but will probably push for it if one day is published becomes something like:
```
["published", "workspace].include(agentConfiguration.scope);
```

🙃

Reply:
absolutely agree. As soon as the logic become more complex than just " scope === X" then the functions make sense

Reply:
but IMHO we should wait until that happens